Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1258,6 +1267,9 @@
 /* Action sheet item */
 "SHOW_SAFETY_NUMBER_ACTION" = "Show new safety number";
```

Comment:
I've actually gone the other direction. We mostly lowercase it, as do the other clients.

The exception is button text which we always (should) title case.